### ISSUE TITLE
Command line tool should make it more obvious there is additional functionality displayed with help command

### ISSUE THREAD

**Issue by vasilich6107** [flutter/flutter#60421]
## Steps to Reproduce

1. Run `flutter run` from the terminal window.
2. You'll see the output in the command line
```
Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
```
3. Press `h` from repeating the full message and you'll see a lot more options
```
Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
s Save a screenshot to flutter.png.
b Toggle the platform brightness setting (dark and light mode).                                                                                                                                                               (debugBrightnessOverride)
w Dump widget hierarchy to the console.                                                                                                                                                                                                  (debugDumpApp)
t Dump rendering tree to the console.                                                                                                                                                                                             (debugDumpRenderTree)
L Dump layer tree to the console.                                                                                                                                                                                                  (debugDumpLayerTree)
S Dump accessibility tree in traversal order.                                                                                                                                                                                      (debugDumpSemantics)
U Dump accessibility tree in inverse hit test order.                                                                                                                                                                               (debugDumpSemantics)
i Toggle widget inspector.                                                                                                                                                                                     (WidgetsApp.showWidgetInspectorOverride)
p Toggle the display of construction lines.                                                                                                                                                                                     (debugPaintSizeEnabled)
o Simulate different operating systems.                                                                                                                                                                                         (defaultTargetPlatform)
z Toggle elevation checker.
g Run source code generators.
M Write SkSL shaders to a unique file in the project directory.
v Launch DevTools.
P Toggle performance overlay.                                                                                                                                                                                       (WidgetsApp.showPerformanceOverlay)
a Toggle timeline events for all widget build methods.                                                                                                                                                                       (debugProfileWidgetBuilds)
An Observatory debugger and profiler on iPhone 11 is available at: http://127.0.0.1:62067/7nV5OgJn8No=/
```

**Expected results:** Full list of options displayed from the beginning

**Actual results:** Reduced list of the options

<details>
  <summary>Logs</summary>
```
[✓] Flutter (Channel master, 1.20.0-3.0.pre.74, on Mac OS X 10.15.4 19E287, locale en-UA)
    • Flutter version 1.20.0-3.0.pre.74 at /Users/vasilich/IdeaProjects/flutter
    • Framework revision 025463f463 (13 hours ago), 2020-06-26 17:38:26 -0400
    • Engine revision fc0e27210c
    • Dart version 2.9.0 (build 2.9.0-19.0.dev 63cf56d925)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/vasilich/Library/Android/sdk
    • Platform android-29, build-tools 28.0.3
    • ANDROID_HOME = /Users/vasilich/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 46.0.1
    • Dart plugin version 192.8052
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (5 available)
    • SM T720       • 4200bf97cc953600                     • android-arm64  • Android 9 (API 28)
    • iPhone 11     • AB64A2E6-2987-4FCA-89F7-15382F7B2879 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-13-5 (simulator)
    • macOS desktop • macos                                • darwin-x64     • Mac OS X 10.15.4 19E287
    • Web Server    • web-server                           • web-javascript • Flutter Tools
    • Chrome        • chrome                               • web-javascript • Google Chrome 83.0.4103.116

• No issues found!

```
```

</details>

**Comment by TahaTesser** [comment]
Hi @vasilich6107 
Initially shows most useful commands IMO,  it makes sense to have more options hidden and we can access them whenever we want since long list can take too much space in the logs

Do you think there are important commands being hidden here?
Thank you

**Comment by vasilich6107** [comment]
Hi @TahaTesser!

I understood your point. I did not start flutter from terminal very often. I use Android Studio.
Lately when I dig through the closed PRs in June I discovered there there was a new `b` key added for switching the brightness.
I tried to check this feature in terminal. There was no mentioning of the `b` key but it worked) I accidentally pressed `h` and was surprised to find a whole new world of commands. I never realized that there is more commands)
I assume that this confusion caused by `h Repeat this help message.` text. I did not expect to sees more than initial message as per command description.
It could be more user friendly to update it to `h Display the full list of available commands`.

**Comment by jonahwilliams** [comment]
I don't think we're going to more commands to the initial screen - we don't want to overwhelm new users. As for the new functionality, we do need to advertise it better. fyi @csells for some thoughts on how we could introduce users more gradually to the full commands.

I like the idea of including "Display the full list of available" or something similar to hint that there is more functionality.

**Comment by TahaTesser** [comment]
Hi @vasilich6107 
This is now fixed by https://github.com/flutter/flutter/pull/80903
`h` command message now clearly indicates there are additional commands
`-h List all available interactive commands.`[line](https://github.com/flutter/flutter/pull/80903/files#diff-6616d3ff60cdec5e723f489544da5fcfcc5bd69ac567b3844bb595bd8d271a22R112)

### Latest `master` 
```console
[  +16 ms] Connected to _flutterView/0xdc6ae010.
[  +11 ms] Flutter run key commands.
[   +3 ms] r Hot reload. 
[   +1 ms] R Hot restart.
[   +2 ms] h List all available interactive commands.
[   +2 ms] d Detach (terminate "flutter run" but leave application running).
[   +5 ms] c Clear the screen
[   +3 ms] q Quit (terminate the application on the device).
[   +6 ms] Running with unsound null safety
[   +6 ms] For more information see https://dart.dev/null-safety/unsound-null-safety
[   +5 ms] An Observatory debugger and profiler on SM M025F is available at: http://127.0.0.1:50619/4EmPNdBUuyg=/
```



Based on https://github.com/flutter/flutter/issues/60421#issuecomment-653146558, I would consider as fixed 
If you disagree, please write in the comments and I will reopen it.
Thank you
